docs/rest_api.rst

```diff
@@ -575,6 +577,7 @@ and/or ``to_block`` to query only a events from a limited range of blocks.
 
    :statuscode 200: For successful query
    :statuscode 400: If the provided query string is malformed
+   :statuscode 409: If the given block number argument is invalid
```

Comment:
Why a [409 Conflict](https://httpstatuses.com/409)?

Reply:
Cause that's what we use for invalid input so far. e.g. for `InvalidAddress`, `InvalidSettleTimeout`, `AddressWithoutCode`, `InvalidAmount` e.t.c.